modules/docs/arrow-docs/docs/docs/integrations/retrofit/README.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: docs
+title: Retrofit
+permalink: /docs/integrations/retrofit/
+---
+
+## Retrofit
+
+{:.advanced}
+advanced
+
+Arrow contains a integration module for Retrofit so you can have a Arrow integration in your endpoints definition.
```

Comment:
so you can use any synchronous or asynchronous datatype of your choice, like `Try`, `ObservableK`, `IO` or `DeferredK`.